### ISSUE TITLE
Load the SQL into the database and Learn the API for the Ruby MySQL gem

### ISSUE THREAD

**Issue by thewoolleyman** [jahjahjah/R000by#3]
PRs #1 and #2 are a prerequisites for this.

First, load the tables.sql file into mysql which will create your database tables and data.  You can google how to do this.

If you aren't familiar with SQL, google for mysql tutorials, and pick any beginner one that looks good to you, to learn basic syntax for selecting data out of the tables. 

Then, Install the [Mysql2](https://github.com/brianmario/mysql2) Ruby Gem, and read the README to learn how to do some simple SQL queries using the gem.  You can run `irb` to do this interactively in Ruby (you'll need to `require "mysql2"` to use load the gem under `irb`).

**Comment by jahjahjah** [comment]
Created database "mini" and tables "users" and "orders"

To install the gem file I was getting permissions error, but used sudo command as workaround. 

Next, I need to experiment using sql with the gem.

**Comment by thewoolleyman** [comment]
@jahjahjah One note - you only had to do sudo because you were using the "system" ruby that comes with OSX (I'm assuming you're on OSX).

It's better to install your own ruby from scratch, as your local user.  That way, it will be writeable and not need sudo, plus you can use the latest version (the system one is outdated).

There's several ways to do this.  One of the easiest is RVM: https://rvm.io/

Alternately, you could use https://github.com/postmodern/ruby-install - but it's a little more bare bones and may require more manual configuration.  Some people prefer that.  RVM will do everything for you.

**Comment by jahjahjah** [comment]
Hmmm. Well, I did install it from scratch. But at some point in the process I messed with the root user / password and was attributing the permissions error to that. I will try to reinstall via RVM method.

**Comment by thewoolleyman** [comment]
@jahjahjah Yes, if you do anything with the root user (which you shouldn't need to if you use RVM or ruby-install), it will make stuff root-owned.

you can either blow that away (via sudo) and start over, or change all the ownership if you can find the files, with `chown -R YOURUSER:YOURUSER the_dir_tree`.  `man chown` for more details.
